tests/datasets/test_dataset.py

```diff
@@ -1255,55 +1238,6 @@ def random_uuids(num_records: int):
             Schema(timestamp_column_name="timestamp", prediction_id_column_name="prediction_id"),
             id="iso8601_us_pacific_strings_converted_to_utc_timestamps",
         ),
-        pytest.param(
-            DataFrame(
-                {
-                    "timestamp": [
-                        "24-03-2023 10:00:00 UTC",
-                        "24-03-2023 11:30:00 UTC",
-                        "24-03-2023 14:15:00 UTC",
-                    ],
-                    "prediction_id": [1, 2, 3],
-                }
-            ),
```

Comment:
@pbadhe Is this a typical timestamp format? It's not one familiar to me. I'm not sure we need to support.